src/LOCATION

```diff
@@ -1 +1 @@
-DIR="/var/ossec"
+DIR="/var/ossec31"
```

Comment:
Please restore the original value of this parameter. This is the default installation directory, it should still being `/var/ossec`.